packages/vue/global-spec.ts

```diff
@@ -19,6 +20,7 @@ export const components = {
   BaseHeading,
   BaseInput,
   BaseLabel,
+  BaseSelect,
```

Comment:
Is this a related change?

Reply:
This was missing from our global-spec file, which was causing several warnings in the Vue tests. I added this in so the tests would not show those warnings. It is related in the sense when I added a new spec file test in, it was causing issues.